dependencies.yaml

```
@@ -295,6 +307,11 @@ dependencies:
               - *libcurand114
       - output_types: conda
         matrices:
+          - matrix:
+              cuda: "12.0"
+              arch: x86_64
+            packages:
+              - libcufile-dev
```

Should this have `cuda-version` as well?

```suggestion
              - cuda-version=12.0
              - libcufile-dev
```

---

That dependency is already supplied by this line: https://github.com/bdice/cudf/blob/6e81e2eb984aaa50cfc1e6fa88c263b5463ef611/dependencies.yaml#L275

The libcufile dependencies are in a separate `- output:` entry because their matrices depend on `cuda` _and_ `arch` (libcufile is not available for ARM).